src/en/readcomic/src/eu/kanade/tachiyomi/extension/en/readcomic/Readcomic.kt

```diff
@@ -0,0 +1,199 @@
+        GET(
+            "$baseUrl/advanced-search".toHttpUrl().newBuilder().apply {
+                addQueryParameter("key", query)
+                if (!filters.isEmpty()) {
+                    for (filter in filters) {
+                        when (filter) {
+                            is StatusFilter -> {
+                                addQueryParameter("status", arrayOf("", "CMP", "ONG")[filter.state])
+                            }
+
+                            is GenreList -> {
+                                addQueryParameter("wg", filter.included.joinToString(","))
+                                addQueryParameter("wog", filter.excluded.joinToString(","))
+                            }
+
+                            else -> {}
+                        }
+                    }
+                }
+            }.build(),
+            headers,
+        )
+
+    override fun popularMangaFromElement(element: Element): SManga =
+        SManga.create().apply {
+            setUrlWithoutDomain(element.selectFirst("a")!!.attr("abs:href"))
+            title = element.selectFirst("h3")!!.text()
+            thumbnail_url = element.selectFirst(".image")!!.selectFirst("img")!!.attr("abs:src")
+        }
+
+    override fun latestUpdatesFromElement(element: Element): SManga =
+        SManga.create().apply {
+            setUrlWithoutDomain(element.attr("abs:href"))
+            title = element.text()
+        }
+
+    override fun searchMangaFromElement(element: Element): SManga {
+        return popularMangaFromElement(element)
+    }
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        val infoElement = document.selectFirst(".manga-details tbody")
+
+        title = infoElement!!.selectFirst("td:contains(Name:)")!!.nextElementSibling()!!.text()
+        author = infoElement.selectFirst("td:contains(Author:)")!!.nextElementSibling()!!.text()
+        genre = infoElement.selectFirst("td:contains(Genre:)")!!.nextElementSibling()!!.text()
+        status = infoElement.selectFirst("td:contains(Status:)")!!.nextElementSibling()!!.text()
+            .orEmpty().let { parseStatus(it) }
+        description = document.select(".pdesc").first()!!.text()
+        thumbnail_url = document.select(".manga-image img").first()!!.attr("abs:src")
+    }
+
+    private fun parseStatus(status: String) = when {
+        status.contains("Ongoing") -> SManga.ONGOING
+        status.contains("Completed") -> SManga.COMPLETED
+        else -> SManga.UNKNOWN
+    }
+
+    override fun chapterListSelector() = ".ch-name"
+
+    override fun chapterFromElement(element: Element): SChapter = SChapter.create().apply {
+        setUrlWithoutDomain(element.attr("abs:href") + "/full")
+        name = element.text()
+        date_upload = element.nextElementSibling()!!.text().let {
+            SimpleDateFormat("MM/dd/yy", Locale.getDefault()).parse(it)?.time ?: 0L
+        }
```

Comment:
wrap in try Catch. also place `SimpleDateFormat` object in the class so as to not initialize it on every chapter